### PR TITLE
M3-3838 Use base-2 for graph metrics

### PR DESCRIPTION
**Pull request by johnwcallahan**
## Description

Use base-2 for graph metrics

Previously the metrics displayed under each Linode and NodeBalancer graph were in base-10. This makes it so that these metrics are calculated with base-2 since that’s the way the data is formatted in the actual graph.

I also changed the following units:

`Kb` -> `Kibit`
`Mb` -> `Mibit`

This affects the Linode graphs and the Longview graphs.

## Type of Change
- Non breaking change ('update', 'change')

## Note to Reviewers

Please test against production and Classic. Compare values for Max, Avg, and Last. The values in this PR should be slightly lower (the difference between base-10 and base-2).

Please remember to test the NodeBalancer graphs (which haven’t actually been updated to use native legends yet).

### CODE REVIEW

File: packages/manager/src/features/Longview/shared/utilities.ts [hunk 1]
```diff
@@ -315,21 +315,21 @@ export const generateNetworkUnits = (networkUsed: number): NetworkUnit => {
...
 };
 
 export const convertNetworkToUnit = (
   valueInBits: number,
   maxUnit: NetworkUnit
 ) => {
-  if (maxUnit === 'Mb') {
+  if (maxUnit === 'Mibit') {
     // If the unit we're using for the graph is Mb, return the output in Mb.
     const valueInMegabits = valueInBits / 1024 / 1024;
```
Comment (acourdavault) [inline packages/manager/src/features/Longview/shared/utilities.ts]: valueInBits >> 20

File: packages/manager/src/features/Longview/shared/utilities.ts [hunk 2]
```diff
@@ -315,21 +315,21 @@ export const generateNetworkUnits = (networkUsed: number): NetworkUnit => {
...
     return valueInMegabits;
-  } else if (maxUnit === 'Kb') {
+  } else if (maxUnit === 'Kibit') {
     // If the unit we're using for the graph is Kb, return the output in Kb.
     const valueInKilobits = valueInBits / 1024;
```
Comment (acourdavault) [inline packages/manager/src/features/Longview/shared/utilities.ts]: value >>10

File: packages/manager/src/utilities/statMetrics.tsx [hunk 3]
```diff
@@ -45,42 +46,10 @@ export const getMetrics = (data: number[][]): Metrics => {
...
-    : `${formatNumber(num)} ${unit}`;
-};
-
 export const formatPercentage = (value: number) => formatNumber(value) + '%';
-export const formatBitsPerSecond = (value: number) =>
-  formatMagnitude(value, 'b/s');
-export const formatBytes = (value: number) => formatMagnitude(value, 'B');
+
+export const formatBitsPerSecond = (valueInBits: number) =>
+  formatNetworkTooltip(valueInBits / 8) + '/s';
```
Comment (acourdavault) [inline packages/manager/src/utilities/statMetrics.tsx]: I would argue this function could be in Longview/shared/utilities.ts right next to `formatNetworkTooltip` as one add the "unit" and one the "/s" and both are "formatting utilities"

counter argument to this is that by putting here this we reduce the scope.
Up to you

File: packages/manager/src/features/Longview/shared/utilities.ts (hunk 1 above)
Comment (johnwcallahan) [reply packages/manager/src/features/Longview/shared/utilities.ts]: I'd argue that this is less clear as it assumes familiarity with bitwise operators (which are not used anywhere in the codebase). We event have a tslint rule against it:

Screen Shot 2020-01-29 at 11 15 31 AM

File: packages/manager/src/features/Longview/shared/utilities.ts (hunk 2 above)
Comment (johnwcallahan) [reply packages/manager/src/features/Longview/shared/utilities.ts]: Same as above.

File: packages/manager/src/utilities/statMetrics.tsx (hunk 3 above)
Comment (johnwcallahan) [reply packages/manager/src/utilities/statMetrics.tsx]: Good suggestion; I just pushed up this change.

File: packages/manager/src/features/Longview/shared/utilities.ts (hunk 1 above)
Comment (acourdavault) [reply packages/manager/src/features/Longview/shared/utilities.ts]: Interesting, ok.

**Pull request merged**